controllers/reconcilers/configuration/token_refresher.go

```diff
@@ -160,6 +160,9 @@ func (r *Reconciler) createDeploymentFor(ctx context.Context, cr *v1.Observabili
 						"app.kubernetes.io/name":      config.Name,
 						"app.kubernetes.io/version":   TokenRefresherImageTag,
 					},
+					Annotations: map[string]string{
+						"cluster-autoscaler.kubernetes.io/safe-to-evict": "true",
```

Comment:
@vmanley is the token refresher using local storage (e.g. an empty dir volumen mount)?

Reply:
No storage used. I'll remove annotations. Thanks